charts/cinder-csi-plugin/templates/controllerplugin-deployment.yaml

```diff
@@ -1,21 +1,17 @@
-apiVersion: v1
-kind: Service
-metadata:
-  name: {{ include "cinder-csi.name" . }}-controllerplugin
-  labels:
-    {{- include "cinder-csi.controllerplugin.labels" . | nindent 4 }}
-spec:
-  clusterIP: None
----    
```

Comment:
um... what's the reason for removing this service?

Reply:
This headless service is not used at all, however it's a best practice to have it for statefulSets and some linters validate that each sts must have a headless service. However if we want to change it to deployment, it would be unnecessary and unused.

Reply:
ok, understood, thanks for the info